man/docker-run.1.md

```diff
@@ -446,12 +446,13 @@ according to RFC4862.
 
    * `src`, `source`: mount source spec for `bind` and `volume`. Mandatory for `bind`.
    * `dst`, `destination`, `target`: mount destination spec.
-   * `ro`, `read-only`: `true` or `false` (default).
+   * `ro`, `readonly`: `true` or `false` (default). Note that setting `readonly` for a bind mount does not make its submounts read-only. See also `bind-nonrecursive`.
```

Comment:
Maybe a bit abstract but...

It would be nice if at some point the kernel supported recursive, read-only bind mounts and we could just enable it because that's the general expected behavior.
So maybe in the call-out here, we can say that it's dependent on the implementation and *currently* Linux doe not make nested mounts read-only.

Reply:
done